Bottom: 242d20a9264c934d530a2ff585b917ef94c0f248
Top:    fbaa87034d6f8d1ee2c2a16ef8b4c331f4804be3
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-05 12:03:14 -0500

Refresh of summary

---

diff --git a/farm-compile b/farm-compile
index eb934fa..b6c3e46 100755
--- a/farm-compile
+++ b/farm-compile
@@ -30,9 +30,8 @@ for bintree in $farmbin/*; do
 	$make clean KERNELRELEASE=0 >/dev/null
 	echo -n "Checking $bintree "
 	if $make HOSTCC="$ucc $ucflags" CC="$kcc $ccflags" \
-	     ARCH=$arch KBUILD=$bintree KERNELPATH=$bintree \
-	     KERNEL_PATH=$bintree C=2 CF="-D__CHECK_ENDIAN__" \
-	     $target >$log 2>&1; then
+	     ARCH=$arch KBUILD=$bintree KERNELPATH=$bintree C=2 \
+	     CF="-D__CHECK_ENDIAN__" $target >$log 2>&1; then
 		mv $log $log-ok
 		echo -e "\e[32mOK\e[0m"
 	else
